enterprise/models.py

```diff
@@ -61,6 +62,8 @@ class Meta(object):
     active = models.BooleanField(default=True)
     history = HistoricalRecords()
 
+    site = models.ForeignKey(Site, related_name="enterprise_customers")
```

Comment:
@asadiqbal08 than probably the [migration is wrong](https://github.com/edx/edx-enterprise/pull/6/files#diff-6695276e9b640f365e80b4987fcd7810R26)